pubsub/topic.go

```diff
@@ -548,6 +548,8 @@ var errTopicStopped = errors.New("pubsub: Stop has been called for this topic")
 //	}
 type PublishResult = ipubsub.PublishResult
 
+var ErrTopicOrderingNotEnabled = errors.New("Topic.EnableMessageOrdering=false, but an OrderingKey was set in Message. Please remove the OrderingKey or turn on Topic.EnableMessageOrdering")
```

Comment:
nit: error message should start with a lower case letter. Also other errors in this package seem to prefix with pubsub. Should we do that here too?

Reply:
As is, this is to maintain complete backwards compatibility (the error message is completely the same in this case). We didn't standardize on prefixing with pubsub.

I also struggled with fixing this for `ErrPublishingPaused`, which starts in the middle of the previous error message to allow for error wrapping.